snapshot/bundlestore/http_store.go

```diff
@@ -62,7 +63,21 @@ func (s *httpStore) Write(name string, data io.Reader) error {
 	}
 	uri := s.rootURI + name
 	log.Printf("Posting %s", uri)
-	_, err := s.client.Post(uri, "text/plain", data)
-	log.Printf("Posted %s, err: %v", uri, err)
-	return err
+	resp, err := s.client.Post(uri, "text/plain", data)
+	log.Printf("Posted %s", uri)
+	if err != nil {
+		log.Printf("Error posting %s, err: %v", uri, err)
+		return err
+	}
+
+	if resp.StatusCode == http.StatusOK {
```

Comment:
Did you merge this from my other PR or add it manually? I remember needing to close resp.Body.